src/UseCases/DisplayPage/PageContentModifier.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend\UseCases\DisplayPage;
+
+use Psr\Log\LoggerInterface;
+
+/**
+ * @licence GNU GPL v2+
+ */
+class PageContentModifier {
+
+	private $logger;
+	private $substitutions;
+
+	public function __construct( LoggerInterface $logger, array $substitutions = [] ) {
+		$this->logger = $logger;
+		$this->substitutions = $substitutions;
```

Comment:
Not done yet as its not clear if we need this here. There is a TODO about it further down. The empty check does not appear to cause a notice even on strict mode. Given that, do you still think the field should be defined and initialized already?